### PR TITLE
Optimisation: Reduce the semantic ui footprint by removing unused components.

### PR DESCRIPTION
**Pull request by microsoftsam**
Reduced from 
1,278,007 bytes (1.3 MB)
to 
1,012,816 bytes (1 MB)

### CODE REVIEW

File: theme/pxtsemantic.less [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+//& { @import "definitions/views/feed"; }
+& { @import "definitions/views/item"; }
+//& { @import "definitions/views/statistic"; }
+
+/* Modules */
+//& { @import "definitions/modules/accordion"; }
+& { @import "definitions/modules/checkbox"; }
+& { @import "definitions/modules/dimmer"; }
+& { @import "definitions/modules/dropdown"; }
+//& { @import "definitions/modules/embed"; }
```
Comment (pelikhan) [inline theme/pxtsemantic.less]: don't we use this to display youtube movies?

File: theme/pxtsemantic.less [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+/* Views */
+//& { @import "definitions/views/ad"; }
+& { @import "definitions/views/card"; }
+//& { @import "definitions/views/comment"; }
+//& { @import "definitions/views/feed"; }
+& { @import "definitions/views/item"; }
+//& { @import "definitions/views/statistic"; }
+
+/* Modules */
+//& { @import "definitions/modules/accordion"; }
```
Comment (pelikhan) [inline theme/pxtsemantic.less]: also remove javascript using this (see modal dialog)

File: theme/pxtsemantic.less [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+& { @import "definitions/modules/dropdown"; }
+//& { @import "definitions/modules/embed"; }
+& { @import "definitions/modules/modal"; }
+//& { @import "definitions/modules/nag"; }
+& { @import "definitions/modules/popup"; }
+//& { @import "definitions/modules/progress"; }
+//& { @import "definitions/modules/rating"; }
+& { @import "definitions/modules/search"; }
+//& { @import "definitions/modules/shape"; }
+& { @import "definitions/modules/sidebar"; }
```
Comment (pelikhan) [inline theme/pxtsemantic.less]: do we use this?

File: theme/pxtsemantic.less (hunk 1 above)
Comment (microsoftsam) [inline theme/pxtsemantic.less]: Where do we display Youtube movies?

File: theme/pxtsemantic.less (hunk 3 above)
Comment (microsoftsam) [inline theme/pxtsemantic.less]: No we dont, I'll remove.

File: theme/pxtsemantic.less (hunk 1 above)
Comment (pelikhan) [inline theme/pxtsemantic.less]: See inchworm project lessons

Comment (pelikhan) [inline theme/pxtsemantic.less]: https://pxt.microbit.org/projects/inchworm

File: theme/pxtsemantic.less (hunk 2 above)
Comment (microsoftsam) [inline theme/pxtsemantic.less]: I'm a little unclear about this, why don't we need to modal? We use it for a number of things in the menu?

Comment (pelikhan) [inline theme/pxtsemantic.less]: Oh accordion Is commented ... my bad.

**Pull request merged**